DOCS-12791 Add governance console overview page (#33335)
DataDog/documentation: content/en/account_management/governance_console/_index.md

- link: "https://www.datadoghq.com/blog/volkswagen-organizations/"
  tag: "Blog"
  text: "Best practices for managing Datadog organizations at scale"
- link: "https://docs.datadoghq.com/account_management/audit_trail/"
  tag: "Documentation"
  text: "Datadog Audit Trail"
- link: "https://docs.datadoghq.com/cloud_cost_management/"
  tag: "Documentation"
  text: "Cloud Cost Management"
---

## Overview

Governance Console unifies the configuration and administration of a large, multi-team, multi-organization Datadog account. For organizations who need centralized governance, strong defaults, and automatic policy enforcement, Governance Console helps establish and maintain a best practice, compliant environment.

Governance Console helps teams understand how their organization is adopting and using Datadog. It provides a centralized, self-service view of Datadog usage and adoption. Governance Console offers control and predictability over the platform's configuration, usage, and cost, helping administrators holistically manage their Datadog deployment. Datadog turns your organization's configuration and usage activity into metrics and insights to guide optimization and investment.

> Governance Console supports both new and existing customers, helping new customers start from a well-managed setup, and helping existing customers mature and adopt best practices over time.

Reviewer: I think this overview section is much too long, a bit repetitive, does too much selling/persuading, and too vague / isn't sufficiently concrete. I think the opening sentence really needs to describe what Governance Console literally is (a page/section in Datadog that displays particular information). We can then offer some brief justification for why you'd want to use it and what specific use-cases it would be most helpful for. I'm also really struggling with the last sentence about how it supports both new and existing customers.... shouldn't all products support new and existing customers?

{{< img src="account_management/governance_console/governance-console-overview.png" alt="Governance console screenshot, showing summary" style="width:100%;" >}}

## Explore usage insights

In the total org usage section, Governance Console displays your organization's use and adoption of Datadog, such as active users and time spent in the product.

Governance Console also exposes per-product insights to learn more about usage and configuration trends on key platform features and products. You can learn about your adoption and maturity, and identify interesting trends or adoption behaviors.

{{< img src="account_management/governance_console/log-management-product.png" alt="Governance console screenshot, showing Log Management metrics" style="width:100%;" >}}

> ## Establish guardrails

Reviewer: Is there a more concrete way we can phrase this? I find this heading to be a little vague.

Also, if this is about the Controls tab, let's put the word "Controls" in the heading to make this all easier to scan. So something like "Use controls to enforce organization policies"

> Datadog governance controls help you automatically implement policies in your Datadog organization by identifying configuration drift, notifying the accountable users, and enforcing compliance. Each control helps establish and maintain a best practice, compliant environment.

Reviewer: ```suggestion
Datadog governance controls help you automatically implement policies in your Datadog organization by identifying configuration drift, notifying the accountable users, and enforcing compliance. Each control helps establish and maintain compliance with best practices.
```

trying to massage the grammar here— "a best practice, compliant environment" doesn't parse as correct to me, but my suggestion probably isn't ideal either
